.github/workflows/apk-builder.yml

```diff
@@ -0,0 +1,93 @@
+name: APK build
+on:
```

Comment:
Don't add the on push to main functionality. Just let it be a workflow dispatch, it's fine. Otherwise you will use the github runners for nothing most of the time + using artifacts space for nothing too.

Reply:
You are right fixed ✅